doc-surrealdb_versioned_docs/version-1.x/surrealql/functions/geo.mdx

```diff
@@ -102,32 +118,39 @@ null
 
 ## `geo::centroid`
 
-The `geo::centroid` function calculates the centroid between two geolocation points.
+The `geo::centroid` function calculates the centroid between multiple geolocation points.
 
 ```surql title="API DEFINITION"
 geo::centroid(geometry) -> number
 ```
-The following example shows this function, and its output, when used in a [`RETURN`](/docs/surrealdb/surrealql/statements/return) statement:
+The following example shows this function, and its output, when used in a [`RETURN`](/docs/surrealdb/surrealql/statements/return) statement. Note: the doubled square brackets are because the function takes an array of an array to allow for more complex types such as MultiPolygon.
 
 ```surql
 RETURN geo::centroid({
 	type: "Polygon",
 	coordinates: [[
-		[-0.38314819, 51.37692386], [0.1785278, 51.37692386],
-		[0.1785278, 51.61460570], [-0.38314819, 51.61460570],
-		[-0.38314819, 51.37692386]
+		[-0.03921743611083755, 51.88106875736589], -- London
```

Comment:
Maybe write some variables instead of comments. Following the same logic as the previous example.

Reply:
Interestingly that didn't work when I tried it, thanks for the reminder. Going to try to find out why

Reply:
Update: Looks like a bug to me, raised an issue https://github.com/surrealdb/surrealdb/issues/3809